README.md

```diff
@@ -36,7 +36,30 @@ Documentation
 =============
 - [API docs](https://wooga.github.io/atlas-node-release/docs/api/)
 - [Release Notes](RELEASE_NOTES.md)
+- Tasks
+	- [NpmCredentialsTask](#npmcredentialstask)
 
+Usage
+=====
+
+## NpmCredentialsTask
+This task can be used to create a npm configuration file.
+
+**Example Configuration:**
+```groovy
+task example (type:wooga.gradle.node.tasks.NpmCredentialsTask) {
+	npmLogin = 'username:password'
```

Comment:
I would split this up into a `UsernamePassword` Credentials object.
It might be better for you at the moment to set this value as one, but it is not very user friendly.

Reply:
Splitted into `npmUser` and `npmPass`